src/presentation/http/router/note.test.ts

```diff
@@ -1,257 +1,45 @@
-
 import { describe, test, expect } from 'vitest';
 
-import notes from '@tests/test-data/notes.json';
-import noteSettings from '@tests/test-data/notes-settings.json';
-
 describe('Note API', () => {
-  describe('GET note/resolve-hostname/:hostname ', () => {
```

Comment:
You can't delete tests that are prescribed in the main branch. Please restore them - they are important for testing ready-made roots.